python/pyspark/sql/tests/test_utils.py

```diff
@@ -1,3 +1,4 @@
+# encoding: utf-8
```

Comment:
` # -*- encoding: utf-8 -*-`?

Reply:
Will fix. Do you have any other suggestions?

Reply:
I think either way is fine (https://www.python.org/dev/peps/pep-0263/#defining-the-encoding)